docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,63 @@
+*******************************************************
+Stay Under your Quota at /home/
+*******************************************************
+It is vital to stay under your ``/home/<username>`` quota to prevent issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
+Utlilize /scratch and /work Directories
+=======================================================
+For long-term research work storage, ``/work`` should be used. If your PI does not have space in ``/work`` setup, they can request it using the following form https://bit.ly/NURC-NewStorage and can request additional storage using https://bit.ly/NURC-StorageExtension. More details of ``/work`` can be found at :ref:`discovery_storage`. For temporary job files you can utilize /scratch/<username>. Please be mindful of the purge policy of ``/scratch`` which is covered in :ref:`discovery_storage.rst`.
+
+Determine Large Files/Directories in /home/<username>
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your ``/home/<username>`` directory ::
+    
+ du -shc .[^.]* *
+
+which will output the size of each file, directory, and hidden directory in your ``/home/<username>`` space, with the total of your ``/home`` directory being the last line of the output. After determining the large files and directories, you can move them to the appropriate locations such as ``/work`` for research or you can back them up and delete them if they are no longer required.
+
+Conda Environment Best Practices
```

Comment:
The skeleton could be clearer. Perhaps consider restructuring the file to have a Best Practices section, then have subsections "~/.conda" "~/.singularity" etc.,

The headings seem too long and difficult to relate back to the main purpose of the page. Plus, it causes the text on the side-menu index to wrap (i.e., difficult to read). Food for thought.

Reply:
I agree and I applied those changes in the latest revision that I will be pushing soon.